portal/server/api.js

```diff
@@ -856,9 +856,27 @@ const getPreviousUtilisationReportsByBank = async (token, bankId) => {
   }
 };
 
+const getLastestReportByBank = async (token, bankId) => {
+  if (!isValidBankId(bankId)) {
+    throw new Error(`Getting latest report failed for id ${bankId}`);
```

Comment:
Can we put a more specific message here? Something like `Invalid bank ID provided: ${bankId}`.